<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptBuilderMainWindow</class>
 <widget class="QMainWindow" name="ScriptBuilderMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScriptBuilderMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QListWidget" name="listWidgetScriptBuilder">
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="movement">
       <enum>QListView::Snap</enum>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="lblScriptName">
      <property name="text">
       <string>Script Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnRemove">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QListWidget" name="listWidgetBlockOptions">
      <property name="dragDropOverwriteMode">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Snap</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>TEST 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TEST 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TEST 3</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="page0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="Test1Properties_UI" name="page1"/>
      <widget class="Test2Properties_UI" name="page2"/>
      <widget class="Test3Properties_UI" name="page3"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScript_Builder">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuScript_Builder"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Test1Properties_UI</class>
   <extends>QWidget</extends>
   <header>Test1Properties_UI.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Test2Properties_UI</class>
   <extends>QWidget</extends>
   <header>Test2Properties_UI.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Test3Properties_UI</class>
   <extends>QWidget</extends>
   <header>Test3Properties_UI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
